CHANGELOG.md

```diff
@@ -1,328 +1,711 @@
+
+### Changed
+* `tap` -> `tape` + `faucet` (@ralphtheninja)
+
+### Fixed
+* fix `write-random.js`, use `leveldown` instead of `lmdb` (@ralphtheninja)
+* fix `bench/db-bench.js` (@ralphtheninja)
+
+## [1.0.2] - 2015-04-26
+
+### Added
+* add documentation about snapshots (@maxogden)
+
+### Changed
+* update logo and copyright (@ralphtheninja)
+* s/rvagg\/node-/level\// (@ralphtheninja)
+* use `n` instead of `nvm` for working `io.js` support (@ralphtheninja)
+* update to `abstract-leveldown@2.1.0` (@ralphtheninja)
+
+## [1.0.1] - 2015-01-16
+
+### Changed
+* upgrade to `nan@1.5` for `io.js` support (@rvagg)
+
+### Fixed
+* fix `LevelDB` builds for modern gcc versions (@sharvil)
+
+## [1.0.0] - 2014-08-26
+
+### Changed
+* upgrade to `nan@1.3` for Node 0.11.13+ support (@rvagg)
+* upgrade to `LevelDB@1.17.0` (@kesla)
+* allow writing empty values: null, undefined, '', [] and Buffer(0). Entries come out as '' or Buffer(0) (@ggreer, @juliangruber, @rvagg)
+* massive speed up of iterators by chunking reads (@kesla)
+* wrap in abstract-leveldown for consistent type-checking across *DOWNs (@kesla)
+* switch to plain MIT license (@andrewrk)
+
+### Removed
+* remove compile option that borked EL5 compiles (@rvagg)
+
+### Fixed
+* fix clang build (@thlorenz)
+* fix minor memory leaks in options attributes (@ggreer)
+
+## [0.10.6] - 2016-01-07
+
+### Added
+* add `iojs`, `node@4` and `node@5` to travis (@ralphtheninja)
+
+### Changed
+* upgrade to `nan@2.1.x` for `node@4` and `node@5` support (@ralphtheninja)
+
+### Removed
+* remove `node@0.8` from travis (@ralphtheninja)
+
+## [0.10.5] - 2015-05-05
+
+### Changed
+* upgrade to `nan@1.8.x` for `iojs` support (@ralphtheninja)
+
+## [0.10.4] - 2015-02-07
+
+### Changed
+* bumped version (@rvagg)
+
+## [0.10.3] - 2015-02-07
+
+### Changed
+* upgrade to `nan@1.5` (@mcollina)
+
+## [0.10.2] - 2013-11-30
+
+### Fixed
+* apply fix by @rescrv for long-standing [OSX corruption bug](https://groups.google.com/forum/#!topic/leveldb/GXhx8YvFiig) (@rvagg / @rescrv)
+
+## [0.10.1] - 2013-11-21
+
+### Changed
+* upgrade to `nan@0.6` for Node@0.11.6 support, v8::Local<T>::New(val) rewritten to NanNewLocal<T>(val) (@rvagg)
+
+**Historical Note** From this release and onward, tags in git start with the prefix `v`, i.e. this release corresponds to the tag `v0.10.1`.
+
+## [0.10.0] - 2013-11-18
+
+### Added
+* add new simple batch() leak tester (@rvagg)
+
+### Changed
+* upgrade to `Snappy@1.1.1` (@rvagg and @No9)
+* upgrade to `nan@0.5.x` (@rvagg)
+* upgrade to `abstract-leveldown@0.11.x` for testing (@rvagg)
+* switch all `callback->Call()`s to `node::MakeCallback()` to properly support Node.js domains (@rvagg)
+* enable LevelDB's BloomFilter (@Kyotoweb)
+* properly enable compression by default (@Kyotoweb)
+
+### Removed
+* remove `Persistent` references for all `batch()` operations as `WriteBatch` takes an explicit copy of the data (@mcollina and @rvagg)
+
+### Fixed
+* fix array-batch memory leak, levelup/#171 (@rvagg)
```

Comment:
Also for later: cleanup links like these.

Reply:
Cleanup as in remove?

Reply:
That was what I was thinking in January but now I'd say standardize them by putting them between parentheses, and run a `remark` preset with `remark-github` against the changelog to auto-link.